DeepNtuplizer/plugins/DeepNtuplizer.cc

```diff
@@ -108,12 +110,31 @@ class DeepNtuplizer : public edm::one::EDAnalyzer<edm::one::SharedResources>  {
   int  Npfcan_isGamma_[100]; 
   float  Npfcan_HadFrac_[100]; 
 
+  // Secondary verticies
+  int n_SV_;
+  float SV_pt_[100];
```

Comment:
Just realized now, can you switch to vectors? The overhead is minimal an in python is much easier to convert (iteration is supported, no risk of running out of bounds)